docs/spec_draft.md

```diff
@@ -518,6 +519,40 @@ returned.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.cbrt
+
+### Semantics
+
+Performs element-wise cubic root operation on `operand` tensor and produces a
+`result` tensor, implementing the `rootn(x, 3)` operation from the IEEE-754
+specification.
```

Comment:
Do we want to say something about numeric precision, similarly to what we did for `rsqrt`?